src/main/java/seedu/address/model/person/Person.java

```diff
@@ -71,6 +141,11 @@ public boolean isSamePerson(Person otherPerson) {
                 && otherPerson.getStudentId().equals(getStudentId());
     }
 
+    public Person getCopy() {
```

Comment:
Might be good if we can include a JavaDoc header here

Reply:
added JavaDocs comment! 👍